clients/js/src/ChromaClient.ts

```diff
@@ -46,7 +47,7 @@ export class ChromaClient {
   /**
    * @ignore
    */
-  private api: DefaultApi & ConfigOptions;
+  public api: DefaultApi & ConfigOptions;
```

Comment:
this has to be public for the Collection object to use it. 

we could pass the api object to `Collection` - but i like the idea that if you change some client settings, that all Collection objects created through that client will start using these settings.

Reply:
I worry a little that this could confuse users who see this pop up in their autocomplete, but I don't think there's a great solution besides making a third wrapper class over `api`. Not a blocking issue.

Reply:
yea - there is a way to pseudo-hide it using some typescript f****ery - but I think that is worse and less maintainable. what about renaming it to something like `_InternalApi`?

Reply:
yeah underscoring seems like a good option for now